src/hlsBinaries.ts

```diff
@@ -346,10 +384,15 @@ export async function getProjectGHCVersion(
                     logger.error(`stdout: ${stdout}`);
                 }
                 // Error message emitted by HLS-wrapper
-                const regex = /Cradle requires (.+) but couldn't find it/;
+                const regex = /Cradle requires (.+) but couldn't find it|The program \'(.+)\' version .* is required but the version of.*could.*not be determined|Cannot find the program \'(.+)\'\. User-specified/;
```

Comment:
This is the regex that will cause `MissingToolError` and matches on various outputs from the wrapper.

Reply:
Ok, this is a work-around until https://github.com/haskell/haskell-language-server/pull/2591 lands?

Reply:
I guess